build.gradle

```diff
@@ -3,10 +3,10 @@
 buildscript {
 
     ext {
-        kotlin_version = '1.9.20'
```

Comment:
why not `1.9.23`?

Reply:
It would probably be fine but I’m just matching to the Kotlin version in Anvil to be safe: https://github.com/square/anvil/pull/814

Reply:
Thanks, was wondering if there was something like that but couldn't find it. Makes sense to me.